### ISSUE TITLE
INSTALL.sh problem: link to lib.zip leads to 404

### ISSUE THREAD

**Issue by dvdkouril** [ryanbartley/Cinder-Bullet3D#8]
Hi,
I'm not sure if I'm doing anything wrong but I wanted to try and use your block. INSTALL.sh script didn't install the libraries properly, the zip file that it downloads seems to be broken (can't be unzipped). I tried to copy the URL into the browser and download the archive that way but it's showing me a 404 page. Is the zip file gone or am I doing something wrong?
Thanks,
David.

**Comment by ryanbartley** [comment]
Hi,

My block for Cinder-Bullet3D is in a bit of flux right now. Basically, I'm getting rid of the pre-built binaries and using the build system that's a part of Bullet3D's repo. It would be great if you could test this. [This](https://github.com/ryanbartley/Cinder-Bullet3D/tree/UpdateSubmodule) is the submodule that I'm going to merge into master soon. And the readme has instructions on the install. If you can try this, that would be extremely helpful in moving forward. Let me know if you have any problems with it.

Best,
Ryan

**Comment by dvdkouril** [comment]
Thanks for answering! 
Looks like I made it work with you instructions. I just had two little problems (which probably come from me using git in a very casual way as a code backup and not much more):
1. your repo has to be cloned, 'git submodule update --init' command won't work if you download the repo as a zip. This is probably logical but it maybe could be mentioned for users like me
2. I had to setup SSH key for git/github, otherwise the submodule doesn't get cloned (permission denied error).

Another issue was that the samples xcode projects had wrong paths to libraries set up. In Build Settings the paths were "../../../lib/macosx/libBulletSomething_x64_debug.a" while the libraries that I compiled were "../../../libmacosx/Debug/libBulletSomething.a". So I changed that and looks like it compiles (tried the Hello World sample).

Thank you very much for your help and for creating this Cinder block. I'm looking forward to using it!
David.

**Comment by ryanbartley** [comment]
I'll reopen this. Thanks for checking it out. I'll push a change to correct the paths to the compiled libraries in the morning.

**Comment by ryanbartley** [comment]
Alright, so I've updated the projects. However, this branch is working off of the forthcoming android_linux branch of Cinder. So, basically I've fixed the paths of all the projects to find the bullet libs properly. However, you'll have to fix the libcinder paths if you want to use this branch and the update against the master branch of cinder. The android_linux branch should become master very soon.
